portfolio/src/main/webapp/index.html

```diff
@@ -2,17 +2,28 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Sally's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
+      <h1><img src="https://i.imgur.com/ulOq4HE.gif" class="logo" alt="Hi, I'm Sally"></h1>
+      <h2><img src="https://i.imgur.com/zDvfFrh.png" class="title" alt="main picture"></h2>
+      <p>I am a second year student at the University of Toronto
+      <br> with a major in Computer Science and Communications.
+      <br>I enjoy reading, writing, dancing, and designing, 
+      <br> and am currently looking for a summer internship.</p>
+      <h3><a href="">RESUME</a> | <a href="port.html">PORTFOLIO</a></h3>
```

Comment:
Looks like the link for resume is not ready?
Which is fine, just want confirm.

Reply:
Yes, it's not ready yet.